bigquery/indexer.py

```diff
@@ -5,18 +5,21 @@
 import os
 import time
 
+from google.cloud import bigquery
 import pandas as pd
 
 from indexer_util import indexer_util
 
-# Log to stderr.
 logging.basicConfig(
     level=logging.INFO,
     format='%(asctime)s %(filename)10s:%(lineno)s %(levelname)s %(message)s',
     datefmt='%Y%m%d%H:%M:%S')
 logger = logging.getLogger('indexer.bigquery')
 
 ES_TIMEOUT_SEC = 20
+SELECT_FORMAT = 'SELECT * FROM `%s`'
```

Comment:
opt nit - For all 3 of these, especially SELECT_FORMAT, I find it more readable if they're inlined. Up to you.

Reply:
Done.